renovate.json

```diff
@@ -41,16 +41,6 @@
       "depNameTemplate": "com.google.cloud:sdk-platform-java-config",
       "datasourceTemplate": "maven"
     },
-    {
```

Comment:
I don't think we want to remove this section all together, we still need it for vertex-ai, dns, notification and grafeas.

Reply:
I changed the config to only update libraries bom version in these 4 modules.

PR title and description are also modified.